docarray/array/array/pushpull/__init__.py

```diff
@@ -0,0 +1,244 @@
+import logging
```

Comment:
can we move the implementation from the __init__.py ? I am not a big fan

Reply:
Ah ok sure